tests/api/subdivisions/test_subdivisions_permissions.py

```diff
@@ -393,3 +394,35 @@ def test_delete(
         url_for("invenio_records_rest.subd_item", pid_value=subdivision["pid"])
     )
     assert res.status_code == 403
+    document.pop("subdivisions")
```

Comment:
Can you keep the original value of subdivisions to restore the data at the end of the test?